install-javafx.html

```diff
@@ -63,11 +63,11 @@ <h2>Run HelloWorld using JavaFX <span class="JFX_MAJOR">11</span></h2>
     </div>
 </div>
 
-<p>
-    <b>Note:</b> Additional modules are required for extended functionality.
+<div class="alert">
+    <strong>Note:</strong>
+    Additional modules are required for extended functionality.
     For example, if your application is using <kbd>FXML</kbd>, you will need to
     add the <kbd>javafx.fxml</kbd> module, as shown below:
-</p>
 
 <ul class="nav nav-tabs">
```

Comment:
While I still think that the tabs should be part of the yellow alert, that leads to a small issue: while the code background is now transparent, as we don't want black over yellow, the tabs background is black, and it doesn't look good. We need new css for this case...

Reply:
Since this is a special case `note section with code` which requires a new css another possibility is to rewrite this part so that no `note box` is needed -> the code section can then be as it is.

And like the outsource from the refactoring from the headers in a new issue we can do here the same. The only changes in the file 'install-javafx.html' is this note section with its code part. So reset the changes for 'install-javafx.html' to the old state (and make no changes in ths PR) and create an new issue would have the advance that we can finished this request and have more time for the rewrite or css from the notified section in the file 'install-javafx-html'. 

What would you preferred?